pkg/injection/codemodule/installer/job/bootstrapper.go

```diff
@@ -101,3 +89,37 @@ func (inst *Installer) buildArgs(jobName, targetDir string) []string {
 		"--work=" + inst.props.PathResolver.AgentJobWorkDirForJob(jobName),
 	}
 }
+
+func (inst *Installer) createCorrectSecurityContext() *corev1.SecurityContext {
+	var sc *corev1.SecurityContext
+
+	if inst.isOpenShift {
+		sc = &corev1.SecurityContext{ // TODO: Use the SecurityContext from the `provisioner` container
```

Comment:
it IS 'provisioner' security context  so 'TODO' could be removed (or the whole comment)